knowledge-base/grid-filterrow-bool-customization.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Change the Boolean Filtering Options in a FilterRow Dropdown
+description: Learn how to customize the default text in a Grid FilterRow dropdown list, including changing "is true" and "is false" to "Yes" and "No," modifying the FilterRow dropdown list display names, and filtering the boolean column values by null for a more user-friendly and tailored filtering experience.
+type: how-to
+page_title: How to Change the Boolean Filtering Options in a FilterRow Dropdown
+slug: grid-kb-filterrow-bool-customization
+ticketid: 1575619, 1603347, 1646267, 1654640
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+This KB article answers the following questions:
+
+* How to change default text `is true` and `is false` in Grid `FilterRow` dropdown?
+* How to change the default display names in `FilterRow` dropdown?
+* How to filter a boolean `Grid` column values by `null`?
+* How to change the default values in `FilterRow` dropdown to "**Yes**" and "**No**"?
+
+## Solution
+
+* Define a [Filter Row Template]({%slug grid-templates-filter%}#filter-row-template).
```

Comment:
Is there no particular order in this list? Can you use an ordered list instead?

Reply:
There is no particular order here. The actions can be implemented in various orders. Currently, the order of the actions can be displayed in an ordered and unordered list. If the ordered list is the preferred one, I will change the list type.